ddtrace/tracer/option_test.go

```diff
@@ -158,3 +158,75 @@ func TestTracerOptionsDefaults(t *testing.T) {
 		assert.Equal(nil, dVal)
 	})
 }
+
+func TestVersion(t *testing.T) {
```

Comment:
Sorry Kyle but this is not what I mean. I meant to say that testing that `DD_SERVICE` gets set appropriately belongs in `option_test.go`, the rest of the tests are related to starting spans and automatic tag setting so those were in the right place.

Reply:
I see. I will move this back then. I removed the test specifically around `DD_SERVICE` since that's tested in option_test.

I will also remove the asserts that check that `span.Service` is what we expect. I think that's what you're meaning.